### DISCUSSION TITLE
💡 Feature Request: optional argument in /location

### DISCUSSION THREAD

**Discussion by Dx8pi** [DiscordDungeons/Bugs#1326] [Ideas]
### Is your feature request related to a problem?

It's sometimes annoying when you wanna see the location info of a place, but don't wanna travel there. I suggest adding an optional argument for /location, where you can enter the name of the location you wanna see, and the info for it will pop up as if you were at that place doing the command. It'll save a lot of travel time and you wont get distracted from the game waiting to get there. Additionally, if the user doesn't have access to it, or even better, hasn't visited the location yet, an error message could pop up saying "You don't have access to this location yet" or "You've yet to visit this location"

**Comment by Sven65** [comment]
@mount2010 Thoughts?
